besu/src/main/java/org/hyperledger/besu/cli/subcommands/networkcreate/mapping/CustomAddressSerializer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.hyperledger.besu.cli.subcommands.networkcreate.mapping;
+
+import org.hyperledger.besu.ethereum.core.Address;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+// TODO Handle errors
+class CustomAddressSerializer extends StdSerializer<Address> {
+
+  CustomAddressSerializer() {
+    this(null);
+  }
+
+  private CustomAddressSerializer(Class<Address> t) {
+    super(t);
+  }
+
+  @Override
+  public void serialize(Address address, JsonGenerator jsonGenerator, SerializerProvider serializer)
+      throws IOException {
+    jsonGenerator.writeString(address.toString());
```

Comment:
(question): isn't it the default behaviour without custom serializer ?

Reply:
No, it serialises the whole object and I want only the address string.
If you don't restrict to the address string, ig serialises as :
```
    "address" : {
      "byteArray" : "EzeuAwAcggW3zBkrgAB0VN/HS4Y=",
      "hexString" : "0x1337ae03001c8205b7cc192b80007454dfc74b86",
      "empty" : false,
      "zero" : false,
      "arrayUnsafe" : "EzeuAwAcggW3zBkrgAB0VN/HS4Y="
    },
```
instead of :
```
"address" : "0x1337ae03001c8205b7cc192b80007454dfc74b86",
```